### PR TITLE
Resize fig4

### PR DESCRIPTION
**Pull request by estherrolf**
fig 4 --> 3.5in x 6.1in

### CODE REVIEW

File: api_keys.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (bolliger32) [inline api_keys.json:1]: Can we remove this file? We don't use the census api anymore (plus don't want to commit any API keys :) )

Comment (estherrolf) [reply api_keys.json:1]: yeah i thought that was weird -- git wouldn't let me pull from master b/c of a conflict but I can delete

**Comment by estherrolf** [comment]
deleted api_keys.json and also changed the way we output data so it's cut off at a reasonable precision -- Hannah was having an issue with the tests failing b/c of rounding issues

**Comment by bolliger32** [comment]
yay! thanks!

**Pull request merged**